codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -402,6 +403,15 @@ public Http2Stream resetSent() {
         }
 
         @Override
+        public Http2Stream headersSent() {
+            headersSent = true;
+            return this;
+        }
+
+        @Override
+        public boolean isHeadersSent() { return headersSent; }
```

Comment:
Ok! Was just following exactly the example set above by the resetSent methods... :)

`        @Override
        public boolean isResetSent() {
            return resetSent;
        }

        @Override
        public Http2Stream resetSent() {
            resetSent = true;
            return this;
        }`